Framework/include/BJet.h

```diff
@@ -143,8 +145,9 @@ class BJet
     }
 
 public:
-    BJet() 
-        : bjets_(nullptr)
+    BJet( std::string myVarSuffix ) 
+        : myVarSuffix_(myVarSuffix)
```

Comment:
Can you add a default value here for "myVarSuffix".

Reply:
Sounds good will do.